chia/full_node/mempool_manager.py

```diff
@@ -599,7 +599,7 @@ async def validate_spend_bundle(
 
         if fail_reason is Err.MEMPOOL_CONFLICT:
             log.debug(f"Replace attempted. number of MempoolItems: {len(conflicts)}")
-            if not can_replace(conflicts, removal_names, potential):
+            if len(conflicts) > 1 or not can_replace(conflicts, removal_names, potential):
```

Comment:
the point of this is to remove the super-set rule as well, right?
in `can_replace()`